### ISSUE TITLE
Looking for new maintainers of go-shopify

### ISSUE THREAD

**Issue by dlebech** [getconversio/go-shopify#108]
***TL;DR: we are looking for new maintainers for go-shopify. Please reach out if you or your organization is interesting in taking the lead :-)***

Even though go-shopify is a fairly small library, I think it is important that the library is well-maintained, given the popularity of both Shopify app development and Golang. Two years after creating go-shopify, it seems that the library is one of the first results when searching for a Shopify Go library which creates certain expectations as well.

However, I am actually not writing any Go code anymore and neither is @getconversio, so it would make sense to get some more knowledgeable people on board that can help shape the future of the library, especially the possible [breaking changes](https://github.com/getconversio/go-shopify/issues/93) that are probably necessary to make the library slightly more useful (and/or modern? :bullettrain_front: )

My old colleagues at Conversio have tried reaching out to @shopify to hear if they would take ownership going forward, but have been unsuccessful in getting a reply so far. We hope that someone in the community, who are actively using go-shopify, will be willing to take over the maintainer hat :tophat: 

Going by the [contributors list](https://github.com/getconversio/go-shopify/graphs/contributors), @jared-fraser @aryy @amwolff @dbertouille @fodawim @RanAbram @davidtonghelix @andrewhoff have all had two or more commits in 2018 (apologies for the direct @-mention :slightly_smiling_face:) so perhaps some of you would be interested in this?

I think the perfect solution would be to hand over the stewardship to another organization with at least two developers that are actively using go-shopify already. I think this might both create some coherence and slightly easier maintenance and communication for new features. I'm not abandoning go-shopify for now by the way. I'll continue to look at pull requests in the near-future.

Thank you! :beers:

**Comment by andrewhoff** [comment]
Hey David (@dlebech)! My colleagues @rickywiens, @dbertouille @markstgodard and I at [Bold Commerce](https://github.com/bold-commerce) would definitely be interested in maintaining for you. Reach out to me at andrew@boldapps.net if you want to talk.

**Comment by dlebech** [comment]
Thanks @andrewhoff ! I'll reach out shortly :+1:

**Comment by amwolff** [comment]
Hi guys,
I had a talk with my colleagues at [datainq](https://github.com/datainq/) today. Considering the work overhead, we agreed that we won't be able to maintain the repository on a daily basis. We're happy to hear that the team at Bold Commerce would like to maintain go-shopify - it's certainly going to be in good hands.

We are definitely going to stay active and contribute to the repository the same way we had done before.

@dlebech Thank you for your amazing work. 🥇🥇🥇

PS:
After the new maintainer is chosen, I suggest to return to the talk about breaking changes as mentioned in #93.

**Comment by dlebech** [comment]
Thanks for weighing in @amwolff, I'm very happy that the repository is being used and contributed to by so many! :heart: 

@stefanosala and @andrewhoff are currently working out the details for maintenance of the repo, so I hope we'll hear something soon :+1:

**Comment by gir** [comment]
Any update on this? While we are doing a lot of development on this, we don't know if we have time to support maintaining this repo.

**Comment by andrewhoff** [comment]
@gir Yes, confirmed that we @bold-commerce will be taking it over. In the next week or two you should see an update here.
